Content.Server/_DV/Abilities/Psionics/FracturedFormPowerSystem.cs

```diff
@@ -149,7 +154,7 @@ private EntityUid GenerateForm(Entity<FracturedFormPowerComponent> original)
             var speciesPrototypes = _prototype.EnumeratePrototypes<SpeciesPrototype>();
             foreach (var proto in speciesPrototypes)
             {
-                if (proto.RoundStart)
+                if (proto.RoundStart && proto.ID != "IPC")
```

Comment:
Hardcoding this feels awful, is there no better way?

Reply:
@Toby222 `SpeciesPrototype` does not provide an easy way to check if the species is Silicon. It's this, or add a vestigial flag to the species prototype.

Reply:
```suggestion
            foreach (var proto in speciesPrototypes)
            {
                var speciesEntityPrototype = _prototype.Index<EntityPrototype>(proto.Prototype);

                if (proto.RoundStart && speciesEntityPrototype.TryGetComponent<PotentialPsionicComponent>(out var canBePsionic, Factory))
                {
                    var chance = canBePsionic.Chance;

                    if (speciesEntityPrototype.TryGetComponent<PsionicBonusChanceComponent>(out var bonusChance, Factory))
                        chance = (chance * bonusChance.Multiplier) + bonusChance.FlatBonus;

                    if (chance > 0)
                        validSpecies.Add(proto.ID);
                }
            }
```
Thing I hammered together, seems to work fine
Generates `Vox, SlimePerson, Diona, Felinid, Human, Reptilian, Chitinid, Moth, Feroxi, Vulpkanin, Oni, Harpy, Thaven, Avali, Kitsune, Rodentia, Arachnid` as potential species list, which seems correct
I believe the intent of the bugfix was to not spawn a non-psionic body, not specifically a non-Silicon one, right?

Reply:
Yeah that should work fine